sklearn/model_selection/tests/test_plot.py

```diff
@@ -548,3 +548,24 @@ def test_learning_curve_display_deprecate_log_scale(data, pyplot):
 
     assert display.ax_.get_xscale() == "linear"
     assert display.ax_.get_yscale() == "linear"
+
+
+@pytest.mark.parametrize(
+    "param_range, xscale",
+    [([5, 10, 15], "linear"), ([-50, 5, 50, 500], "symlog"), ([5, 50, 500], "log")],
+)
+def test_validation_curve_display_accepts_list(pyplot, data, param_range, xscale):
```

Comment:
Here is a tentative more descriptive test name:


```suggestion
def test_validation_curve_xscale_from_param_range_provided_as_a_list(
    pyplot, data, param_range, xscale
):
```

I am not sure that this is valid black though.